explanation/anbox-cloud.md

```diff
@@ -74,3 +74,18 @@ The main components powering the streaming stack in Anbox Cloud are:
 **TURN/STUN servers**: Servers that find the most optimal network path between a client and the container running its application. The streaming stack provides secure STUN and TURN servers, but you can use public ones as well.
 
 **NATS**: A messaging system that the different components use to communicate (see the [project page](https://github.com/nats-io)).
+
+<a name="juju-bundles">
+## Juju bundles
+
+Anbox Cloud provides two different Juju bundles:
+
+* The `anbox-cloud-core` bundle provides a minimised version of Anbox Cloud. This version is sufficient for smaller scale use cases, such as application testing or automation, or if you generally don't want to use the Anbox Cloud streaming stack.
```

Comment:
`This version is sufficient for smaller scale use cases`
This is a bit misleading, both the `core` and `full` bundles can handle the same scale deployments. The `full` bundle is similar to `core`, only with the streaming stack added
The `core` bundle is meant for users who have no need for streaming